packages/headless/src/features/facets/facet-set/facet-set-analytics-actions.ts

```diff
@@ -46,19 +46,6 @@ export const logFacetShowLess = (facetId: string): SearchAction =>
     }
   );
 
-export const logFacetSearch = (facetId: string): SearchAction =>
```

Comment:
That's good that we didn't export it

Reply:
Well if it was exported, I think I would've no-op it, deprecate it, and a console.warn.

Reply:
So, if we compare with what the JSUI is doing, it's indeed not logging a search event.

It is logging a custom event though... 🤔

So I think the actual proper fix is to keep everything else the same, except changing the event type in `coveo.analytics`, and the type on line 52 here to `AnalyticsType.Custom`


Those custom event should then not be causing any incoherent events in UA.

Reply:
edit: After discussion in scrum removing this event is the right thing, so disregard the above.